ember-app/tests/unit/mixins/cards/card-move-test.js

```diff
@@ -0,0 +1,81 @@
+import Ember from 'ember';
+import CardMoveMixin from 'app/mixins/cards/card-move';
+
+import {
+  moduleFor,
+  test
+} from 'ember-qunit';
+
+var sut;
+
+function createIssueData(){
+  return Ember.Object.create({
+    _data: { order: 2.5 },
+    data: Ember.Object.create({ id: 123, title: 'test' })
+  });
+}
+
+function minMaxZeroOrderFixAlgorithm(issue){
+  var order = issue._data.order;
+  while(order < 1e-319){ order *= 10; }
+  while(order > 1e307){ order /= 10; }
+
+  return order;
+}
+
+moduleFor('mixin:cards/card-move', 'CardMoveMixin', {
+  setup: function(){
+    sut = Ember.Object.
+      createWithMixins(CardMoveMixin, {});
+  }
+});
+
+test('move the issue to the top', function(assert){
+  var cardMover = sut.cardMover;
+  cardMover.set('orderKey', 'order');
```

Comment:
In tests I like to set this sort of thing to a never-used-for-real value like `test_order` to make sure we're using `orderKey` everywhere rather than `"order"` on accident.